src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
-}
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 
+  static dellete() {
+    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
+  }
+}
```

Comment:
Here is better to use a more descriptive name for this function.
Example:
```suggestion
  static deleteDeadAnimals() {
    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
  }
}
```